src/sql/workbench/contrib/notebook/browser/htmlMarkdownConverter.ts

```diff
@@ -150,3 +219,20 @@ export function findPathRelativeToContent(notebookFolder: string, contentPath: U
 	}
 	return '';
 }
+
+export function escapeAngleBrackets(textContent: string, node?) {
```

Comment:
Node should have a type

Reply:
And the function should also specify void as the return type

Reply:
I was able to refactor and found I don't utilize node anymore. Will delete.